PyInstaller/hooks/hook-mpl_toolkits.basemap.py

```diff
@@ -8,9 +8,32 @@
 #-----------------------------------------------------------------------------
 
 
-from PyInstaller.utils.hooks import (collect_data_files)
+from PyInstaller.utils.hooks import collect_data_files
+from PyInstaller.compat import is_win
+
+import os, sys
 
 
 # mpl_toolkits.basemap (tested with v.1.0.7) is shipped with auxiliary data,
 # usually stored in mpl_toolkits\basemap\data and used to plot maps
-datas = collect_data_files('mpl_toolkits.basemap')
+datas = collect_data_files('mpl_toolkits.basemap', subdir='data')
+
+# check if the data has been effectively found
+if len(datas) == 0:
+    
+    # - conda-specific
+    if hasattr(sys, 'real_prefix'):  # check if in a virtual environment
```

Comment:
In `PyInstaller.compat` we are using a more sophisticated method to get the `base_prefix`, so we can handle different kinds of virtual environments. Any reason yu are not using `PyInstaller.compat.base_prefix`? Did you test your code with conda + virtualenv, resp. conada + pyvenv?